docs-2.0/3.ngql-guide/6.functions-and-expressions/1.math.md

```diff
@@ -29,8 +29,17 @@ int rand32(int min, int max) | Returns a random 32 bit integer in [min, max).|
 int rand64() | Returns a random 64 bit integer. |
 int rand64(int max) | Returns a random 64 bit integer in [0, max). |
 int rand64(int min, int max) | Returns a random 64 bit integer in [min, max).|
-collect() | Puts all the collected values to a list.
-collect_set() | Puts all the collected values to a set.
-int size() | Returns the number of elements in a list.
-map(fun, iter) | Returns a map object after applying the given function to each item of a given iterable.
-int range(int, int, step) | Returns a list of integers from the starting integer to the end integer in optional steps. The default step is one.
+collect() | Puts all the collected values to a list.|
+collect_set() | Puts all the collected values to a set.|
+avg() | Returns the average value of the argument.|
+count() | Returns the number of records.|
+max() | Returns the maximum value.|
+min() | Returns the minimum value.|
+std() | Returns the population standard deviation.|
+sum() | Returns the sum value.|
+bit_and() | Bitwise AND.|
+bit_or() | Bitwise OR.|
+bit_xor() | Bitwise exclusive OR (XOR).|
+int size() | Returns the number of elements in a list.|
```

Comment:
can size() or length() work on a map?

Reply:
size() can but length() can't. I'll modify the doc.